custom/aaa/dbaccessors.py

```diff
@@ -0,0 +1,336 @@
+
+    def consumables_disbursed(self):
+        return {
+            'ifaTablets': 'N/A',
+            'thrDisbursed': 'N/A',
+        }
+
+    def immunization_counseling_details(self):
+        return {
+            'ttDoseOne': 'N/A',
+            'ttDoseTwo': 'N/A',
+            'ttBooster': 'N/A',
+            'birthPreparednessVisitsByAsha': 'N/A',
+            'birthPreparednessVisitsByAww': 'N/A',
+            'counsellingOnMaternal': 'N/A',
+            'counsellingOnEbf': 'N/A',
+        }
+
+    def abortion_details(self):
+        return {
+            'abortionDate': 'N/A',
+            'abortionType': 'N/A',
+            'abortionDays': 'N/A',
+        }
+
+    def maternal_death_details(self):
+        return {
+            'maternalDeathOccurred': 'N/A',
+            'maternalDeathPlace': 'N/A',
+            'maternalDeathDate': 'N/A',
+            'authoritiesInformed': 'N/A',
+        }
+
+    def delivery_details(self):
+        return {
+            'dod': 'N/A',
+            'assistanceOfDelivery': 'N/A',
+            'timeOfDelivery': 'N/A',
+            'dateOfDischarge': 'N/A',
+            'typeOfDelivery': 'N/A',
+            'timeOfDischarge': 'N/A',
+            'placeOfBirth': 'N/A',
+            'deliveryComplications': 'N/A',
+            'placeOfDelivery': 'N/A',
+            'complicationDetails': 'N/A',
+            'hospitalType': 'N/A',
+        }
+
+    def postnatal_care_details(self):
+        return [{
+            'pncDate': '2019-08-20',
+            'postpartumHeamorrhage': 0,
+            'fever': 1,
+            'convulsions': 0,
+            'abdominalPain': 0,
+            'painfulUrination': 0,
+            'congestedBreasts': 1,
+            'painfulNipples': 0,
+            'otherBreastsIssues': 0,
+            'managingBreastProblems': 0,
+            'increasingFoodIntake': 1,
+            'possibleMaternalComplications': 1,
+            'beneficiaryStartedEating': 0,
+        }]
+
+    def antenatal_care_details(self):
+        return [{
+            'ancDate': 'N/A',
+            'ancLocation': 'N/A',
+            'pwWeight': 'N/A',
+            'bloodPressure': 'N/A',
+            'hb': 'N/A',
+            'abdominalExamination': 'N/A',
+            'abnormalitiesDetected': 'N/A',
+        }]
+
+
+class EligibleCoupleQueryHelper(object):
+    def __init__(self, domain, person_case_id, month_end):
+        self.domain = domain
+        self.person_case_id = person_case_id
+        self.month_end = month_end
+
+    def eligible_couple_details(self):
+        woman = Woman.objects.get(domain=self.domain, person_case_id=self.person_case_id)
+        children_by_sex = Child.objects.filter(mother_case_id=self.person_case_id).aggregate(
+            male=Sum(Case(When(sex='M', then=1), output_field=IntegerField())),
+            female=Sum(Case(When(sex='F', then=1), output_field=IntegerField())),
+        )
+        try:
+            male_children_died = int(woman.num_male_children_died)
```

Comment:
this is from a silly mistake of making these TextFields. Will likely migrate these to integers at a later time